src/qt/bitcoin.cpp

```diff
@@ -420,6 +449,28 @@ int main(int argc, char *argv[])
         Snapshot.DeleteSnapshot();
     }
 
+    // We received a request to remove blockchain data so client user can start to sync from 0
+    if (fSyncfromzeroRequest)
+    {
+        UpgradeQt Syncfromzero;
+
+        // Release LevelDB file handles on Windows so we can remove the old
+        // blockchain files:
+        //
+        // We should really close it in Shutdown() when the main application
+        // exits. Before we can do that, we need to solve an old outstanding
+        // conflict with the behavior of "-daemon" on Linux that prematurely
+        // closes the DB when the process forks.
+        //
+        CTxDB().Close();
+
+        if (Syncfromzero.SyncFromZero(app))
+            LogPrintf("Syncfromzero: Success!");
```

Comment:
Some of the logs message start with `Syncfromzero: ` while others say `Sync from zero: `. Probably should pick one and stay with it. Maybe one of the two or even `Sync From Zero:` or   `SyncFromZero:  `

Reply:
all logging is syncfromzero: its the message pop ups in qt that are spaced.